db/migrate/20160429133701_create_games.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+      t.string :status
+      t.integer :moves
+      t.text :board
```

Comment:
Yes I agree I'm just giving a visualization of what the object would look
like when we get it out of the database. Or am I not understanding?
On Apr 29, 2016 11:36 AM, "Kaoru Kohashigawa" <notifications@github.com>
wrote:

> In db/migrate/20160429133701_create_games.rb
> <https://github.com/kaorus-army/chess_with_friends/pull/2#discussion_r61597164>
> :
>
> > @@ -0,0 +1,11 @@
> > +class CreateGames < ActiveRecord::Migration
> > +  def change
> > +    create_table :games do |t|
> > +      t.string :status
> > +      t.integer :moves
> > +      t.text :board
>
> not sure what you mean? by
> http://www.jsoneditoronline.org/?id=caef066ba81c4f918f20d758bcd7bd11
>
> that's still a json object, we ought to leverage postgres's features
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaorus-army/chess_with_friends/pull/2/files/ad587f63fbd56c5b21eee143a5953e94f6f131fd#r61597164>
>